blogs.json

```
@@ -5261,6 +5261,13 @@
             "feed_url": "https://exyte.com/feed/rss/cocoa-delights",
             "twitter_url": "https://twitter.com/ExyteHQ"
           },
+          {
+            "title": "Code Review | Curated code improvement tips",
```

It should be one of the two:

```suggestion
            "title": " Code Review | Curated code improvement tips",
```
 or 

```suggestion
            "title": "iOS Code Review | Curated code improvement tips",
```

@jeroenleenarts did you have any issues with display of ? I'm honestly not sure if it's accessible enough via different reader and email apps.